frontend/src/container/OptionsMenu/AddColumnField/styles.ts

```diff
@@ -0,0 +1,15 @@
+import { Card } from 'antd';
+import styled from 'styled-components';
+
+export const SearchIconWrapper = styled(Card)<{ $isDarkMode: boolean }>`
+	border-color: ${({ $isDarkMode }): string =>
+		$isDarkMode ? '#424242' : '#d9d9d9'};
```

Comment:
try to colors from antd colors if not found use from [here](src/constants/theme.ts)